### PR TITLE
Add apiPermission method.

### PR DESCRIPTION
**Pull request by krolov**
Hello @balthazar 
I added one new method.
Also fixed some old types

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -203,8 +203,8 @@ declare module 'binance-api-node' {
   }
 
   export interface SetBNBBurnOptions {
-    spotBNBBurn: boolean
-    interestBNBBurn: boolean
+    spotBNBBurn?: boolean | booleanString
+    interestBNBBurn?: boolean | booleanString
```
Comment (balthazar) [inline index.d.ts:207]: what's booleanString?

Comment (krolov) [reply index.d.ts:207]: it's a new type from previous pull request

Comment (krolov) [reply index.d.ts:207]: I don't think it's the best decision but Binance API has a lot of mess about boolean things

Comment (krolov) [reply index.d.ts:207]: it was created here https://github.com/Ashlar/binance-api-node/pull/493

Comment (balthazar) [reply index.d.ts:207]: sounds good

**Comment by balthazar** [comment]
Thanks!

**Pull request merged**